public/app/core/components/Select/DataSourcePicker.tsx

```diff
@@ -67,23 +68,25 @@ export class DataSourcePicker extends PureComponent<Props> {
     };
 
     return (
-      <Select
-        className="ds-picker select-container"
-        isMulti={false}
-        isClearable={false}
-        backspaceRemovesValue={false}
-        onChange={this.onChange}
-        options={options}
-        autoFocus={autoFocus}
-        onBlur={onBlur}
-        openMenuOnFocus={openMenuOnFocus}
-        maxMenuHeight={500}
-        menuPlacement="bottom"
-        placeholder={placeholder}
-        noOptionsMessage="No datasources found"
-        value={value}
-        invalid={invalid}
-      />
+      <div aria-label={selectors.components.DataSourcePicker.container}>
```

Comment:
I've tried to do that but not all of `<SelectBase>` props passed down to ReactSelect. should I modify the SelectBase to handle all props?

Reply:
Not sure @dprokop should these kind of html props be just propagated in all UI components, or is that case by case basis?

Reply:
In any case @zoltanbedi, lets merge this as it is. we can change this later.

Reply:
Yes, this is unfortunate, I've seen this a lot when creating e2e tests but haven't bothered about it because of release crunch time. Creating a div around these components have been my solution as well so far.

Reply:
@hugohaggmark I think some of the UI components just pass any html props down to the element. Like `Button` props are `ButtonProps + HTMLButtonProps`. Probably not possible everywhere and not sure about `style` and `className`, but probably good pattern in general.